app/main.py

```diff
@@ -1 +1,48 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.appetite and self.is_hungry:
```

Comment:
```suggestion
        if self.is_hungry:
```
Not need to check by self.appetite